src/.nuget/Microsoft.NETCore.ILAsm/runtime.Android.Microsoft.NETCore.ILAsm.props

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
@chcosta and @gkhanna79: Still not sure how to achieve what you are suggesting; if I create a `packageTargetOSGroup` for Android without adding the `runtime.Android` files package build fails.  Is there another OS that does this that I could reference?